client/src/component/AuthorInfoSection/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Image, Button, InputGroup, Row, Col, FormControl,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import './AuthorInfoSection.css';
+
+const AuthorInfoSection = (props) => {
+  const [username, setUsername] = useState('');
+  const [fullName, setFullName] = useState('');
+  useEffect(() => {
+    setUsername(props.username);
+    setFullName(props.fullName);
+  }, [props.username, props.fullName]);
+  return (
+    <Row className="justify-content-center AuthorInfoSection ">
+      <br />
+      {props.isLoggedIn ? (
+        <Col lg="4" md="8" xs="8" sm="8" className="text-center">
+          <Image
+            src={props.profilePic}
+            width={100}
+            height={100}
+            alt="Author"
+            className="AuthorImage border border-primary text-center"
```

Comment:
please use consisten style guidelines.

Reply:
> please use consisten style guidelines.

@p-racoon install `stylelint` extension [in your vscode to help with this](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint)

Reply:
> > please use consisten style guidelines.
> 
> @p-racoon install `stylelint` extension [in your vscode to help with this](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint)

@karansapolia I think nimish here is pointing out the class naming conventions. 
fixed it in my next update.